docs/user-guide/further-configuration/browser-context.md

```diff
@@ -0,0 +1,34 @@
+# Browser configuration
+
+`sonarwhal` allows you to define what browsers are relevant to your
+scenarion by adding the property `browserslist` to your `.sonarwhalrc`
+file. This property follows the same convention as [`browserslist`][browserslist]:
+
+```json
+{
+    "browserslist": [
+        "> 1%",
+        "last 2 versions"
+    ]
+}
+```
+
+By specifying this property, you are giving more information to the
+rules and they might decide to adapt their behavior. An example of
```

Comment:
`and they might decide to adapt their behavior`=>` allowing them to adapt their behavior`?